README.md

```diff
@@ -249,6 +249,7 @@ The generic transform module maps the input json keys into another set of keys.
 This allows to perform subsequent operations using a uniform set of keys.
 In some use cases, only a subset of the provided fields are required.
 Using the generic transform, we may specify those particular fields that interest us.
+To include keys that are not specified in the `rules`, add the line `maintain: true`.
```

Comment:
the term `maintain` is not very explanatory ... maybe use `preserveOriginalKeys` 

Also the explanation ... it is not that the fields are not specified in the `rules`, but that those are all fields in the original record ...